doc/classes/PackedByteArray.xml

```diff
@@ -414,7 +414,7 @@
 			<return type="int" />
 			<param index="0" name="new_size" type="int" />
 			<description>
-				Sets the size of the array. If the array is grown, reserves elements at the end of the array. If the array is shrunk, truncates the array to the new size. Calling [method resize] once and assigning the new values is faster than adding new elements one by one.
+				Sets the size of the array. If the array is grown, reserves elements at the end of the array. If the array is shrunk, truncates the array to the new size. Calling [method resize] once and assigning the new values is faster than adding new elements one by one. Returns [constant OK] on success, or one of the other [enum Error] constants if this method fails, use [method size] to find the size of the array.
```

Comment:
```suggestion
				Sets the size of the array. If the array is grown, reserves elements at the end of the array. If the array is shrunk, truncates the array to the new size. Calling [method resize] once and assigning the new values is faster than adding new elements one by one. Returns [constant OK] on success, or one of the other [enum Error] constants if this method fails. Use [method size] to find the actual size of the array after resize.
```
Same for others.

Reply:
I'll do the edit myself and squash the commits, while making things consistent with Array.